packages/scatterplot/src/ScatterPlotItem.js

```diff
@@ -34,6 +58,19 @@ const ScatterPlotItem = ({
     />
 )
 
+const commonProps = {
```

Comment:
you can set this directly on `DefaultSymbol.propTypes`, I used this approach when we have several implementation of the same component (canvas, html, …)